plugins/modules/source_control/gitlab/gitlab_user.py

```diff
@@ -383,6 +402,13 @@ def existsUser(self, username):
             return True
         return False
 
+    '''
+    @param username Username of the user
+    '''
+    def isActive(self, username):
+        user = self.findUser(username)
+        return user.attributes['state'] == 'active'
```

Comment:
```suggestion
        return user.attributes['state'] == 'active'
```
can it fail if user doesn't exist? I think we should add a condition to the main function

Reply:
Using your suggestion to add an extra check to the main function